Nodejs/Product/Nodejs/Workspace/ContextMenuProvider.cs

```diff
@@ -240,6 +295,14 @@ private void WriteNpmOutput(object sender, NpmLogEventArgs e)
                     this.outputPane.WriteLine(e.LogText.TrimEnd('\r', '\n'));
                 }
             }
+
+            private struct BuildProgressUpdater : IProgress<IFileContextActionProgressUpdate>
```

Comment:
>IProgress [](start = 50, length = 9)

There isn't already a no-op implementation?  If not, it should probably be a singleton class.

Reply:
made a readonly field